src/Akka.Persistence.Linq2Db.Compatibility.DockerTests/SqlServer/SqlServerCompatibilitySpecConfig.cs

```diff
@@ -31,15 +31,13 @@ class = ""Akka.Persistence.SqlServer.Snapshot.SqlServerSnapshotStore, Akka.Persi
         linq2db {{
             class = ""{typeof(Linq2DbSnapshotStore).AssemblyQualifiedName}""
             plugin-dispatcher = ""akka.persistence.dispatchers.default-plugin-dispatcher""
-            #plugin-dispatcher = ""akka.actor.default-dispatcher""
             connection-string = ""{DbUtils.ConnectionString}""
-            provider-name = """ + LinqToDB.ProviderName.SqlServer2017 + $@"""
-            table-compatibility-mode = sqlserver
-            tables {{
```

Comment:
FWIW, this 'tables' structure in the HOCON was borrowed from akka-persistence-jdbc. I don't think there is 'much' we don't have compared to the last 'pre-lockdown' commit they have, but it -may- have a small impact on those remaining bits.